pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux_test.go

```diff
@@ -354,7 +358,9 @@ func TestRestart(t *testing.T) {
 		case <-connChan:
 		}
 
-		time.Sleep(time.Second)
+		time.Sleep(dbusReconnectPeriod)
```

Comment:
what is the this sleep required for? If I understand correctly this unit test will take at least 5 seconds, since we iterate over the loop 5 times and sleep for second each time. For unit tests, it's desirable to keep them as fast as possible and avoid sleeps if possible.

Is this sleep needed / any way we can get rid of it?

Reply:
Updated, I reduced the number of tests and sleep time, and compared the code coverage before and after the modification is the same.

Reply:
Nice, thanks!